SR2025/2024-09-16-place-rejection.md

```diff
@@ -0,0 +1,28 @@
+---
+to: Student Robotics 2025 rejections
+subject: Student Robotics 2025 is sadly full
+---
+
+Hi,
+
+We had a lot more interest in Student Robotics 2025 than we had places
+available. This has meant that unfortunately we're unable to offer you a place
+this year.
+
+We realise this is disappointing (we're disappointed too!), however we are
+limited by the number of kits we have and by the size of the venue. We are
+always looking for ways to support more teams and hope to be able to do so in
+future.
+
+We've put you on our waiting list so if we have any early drop outs we will be
+sure to let you know. You may be able to take their place, but we cannot promise
+anything. Let us know if you don't want to be on the waiting list.

```

Comment:
Making this conditional means we can send to both rejected and waiting listed teams
```suggestion
{{ if team_is_on_waiting_list }}
We've put you on our waiting list so if we have any early drop outs we will be
sure to let you know. You may be able to take their place, but we cannot promise
anything. Let us know if you don't want to be on the waiting list.
{{ end_if }}
```